homeassistant/components/ifttt/__init__.py

```diff
@@ -46,18 +48,37 @@
     if DOMAIN not in config:
         return True
 
-    key = config[DOMAIN][CONF_KEY]
+    conf_keys = config[DOMAIN][CONF_KEY]
+    if isinstance(conf_keys, str):
+        conf_keys = {"default": conf_keys}
```

Comment:
Maybe naming it `"api"` would give a less confusing error message when sending fails.

Reply:
To be clear, I was suggesting renaming the `default` here, so the error message does not become `IFTTT reported error sending event to default` which seems a bit gibberish to me.

Just a suggestion though, no biggie :)